packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -430,10 +428,13 @@ export default class LocusInfo extends EventsScope {
         LoggerProxy.logger.warn(
           'Locus-info:index#isMeetingActive --> Call Ended, locus state is inactive.'
         );
-        Metrics.postEvent({
-          event: eventType.REMOTE_ENDED,
-          meetingId: this.meetingId,
+        NewMetrics.submitClientEvent({
+          name: 'client.call.remote-ended',
```

Comment:
we should also remove all the const values from eventType defined in src/metrics/config.ts (like REMOTE_ENDED in this case), because they're not used anymore

Reply:
i have a separate JIRA to do cleanup

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-440534